tshirtsn.cpp

```diff
@@ -1,7 +1,10 @@
 #include <assert.h>
 #include <iostream>
+#include<string>
+#include "./tshirtsn.h"
+using std::string;
 
-char size(int cms) {
+char size_measure(int cms) {

```

Comment:
Good name. Next step would be to try and word it in terms of the translation. How does this sound? `shouldersize_to_tshirtsize`